env-check.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
-mydir=$(cd "$(dirname "$0")"; pwd)
+mydir=$(cd "$(dirname "$0")" || exit; pwd)
```

Comment:
The better way to fix this warning is `set -e`.

This is a bit more work though because `set -e` probably requires something like this too:
```shell
func_check_pkg expect || : $((check_res++))
func_check_pkg aplay || : $((check_res++))
func_check_pkg python3 || : $((check_res++))
func_check_python_pkg graphviz || : $((check_res++))
```

Reply:
Only `SC2143: Use grep -q instead of comparing output with [ -n .. ]` left, I think this is releated to code login improvement, let's do this in a new PR.